firebase-test/package.json

```diff
@@ -13,20 +13,20 @@
   "main": "lib/index.js",
   "dependencies": {},
   "devDependencies": {
-    "@firebase/rules-unit-testing": "^1.3.14",
-    "@types/jest": "^27.0.1",
+    "@firebase/rules-unit-testing": "^1.3.15",
```

Comment:
Looks like we're updating a bunch of dependencies (both run-time, and dev-time), as part of this work. It's not clear to me that the work item itself required the updates, but I suspect not. If not, then is it our normal practice to include dependency updates in commits for work items? I could see an argument for updating the dependencies separately, but that could lead to a bunch of technical debt-only tickets/work . . .

Reply:
For the CLUE application as a whole I generally do a round of dependency updates as a separate PR when preparing new releases. These dependencies aren't for the CLUE application itself, however, but rather for the `firebase-test` folder which is only used for testing firestore security rules. Since security rules change infrequently and it doesn't really make sense to include them with the CLUE application dependency updates I think it makes sense to keep the dependencies up-to-date with individual PRs that touch the rules.